central/globaldb/postgres.go

```diff
@@ -36,7 +36,7 @@ SELECT TABLE_NAME
               , SUM(c.reltuples) OVER (partition BY parent) AS row_estimate
               , SUM(pg_total_relation_size(c.oid)) OVER (partition BY parent) AS total_bytes
               , SUM(pg_indexes_size(c.oid)) OVER (partition BY parent) AS index_bytes
-              , SUM(pg_total_relation_size(reltoastrelid)) OVER (partition BY parent) AS toast_bytes
+              , SUM(COALESCE(pg_total_relation_size(reltoastrelid), 0)) OVER (partition BY parent) AS toast_bytes
```

Comment:
curious if it should be this instead: 

```
COALESCE(SUM(pg_total_relation_size(reltoastrelid)) OVER (partition BY parent), 0) AS toast_bytes
```

I haven't tested it, but I'm curious if they both give the same results or if one is more efficient than the other